app/panel/utils/utils.js

```diff
@@ -232,12 +233,21 @@ export function setTheme(doc, name, account) {
 		const { css } = themeData[name];
 
 		// Create element for the theme being set, if it is not there
-		const themeStyle = doc.createElement('style');
-		themeStyle.id = name;
-		themeStyle.title = `${styleTitlePrefix} ${name}`;
-
-		// Set content of style element to the theme text.
-		themeStyle.textContent = css;
-		document.head.appendChild(themeStyle);
+		if (!themeStyle) {
+			themeStyle = doc.createElement('style');
+			themeStyle.id = name;
+			themeStyle.title = `${styleTitlePrefix}`;
+			themeStyle.textContent = css;
+			doc.head.appendChild(themeStyle);
+		} else {
+			themeStyle.textContent = css;
+		}
```

Comment:
We can simplify by removing the `else` and moving `themeStyle.textContent` below the `if` block

Reply:
@wlycdgr line 241 requires themeStyle it cant be rearranged like this.

Reply:
@fcjr sorry if I hallucinated you commenting on this, but I think we can set `textContent` after appending, no?

Reply:
Simplified, I left the `themeStyle.textContent = css` line inside the if block as is in case there's any funky stuff going on with that content not being applied before it gets appended